lighthouse-core/computed/load-simulator.js

```diff
@@ -63,9 +63,14 @@ class LoadSimulator {
         break;
       default:
         // intentionally fallback to simulator defaults
-        break;
+        return new Simulator(options);
     }
 
+    // 0 should be interpreted as "unset" for these values.
+    // Fallback to environment values in this case.
+    if (options.rtt === 0) options.rtt = networkAnalysis.rtt;
```

Comment:
I assumed this is true for `rtt` but it shouldn't be necessary to fix the issue at hand.